custom_metrics/almanac.js

```diff
@@ -307,5 +335,248 @@ return JSON.stringify({
     }
   })(),
   //  check if there is any picture tag containing an img tag
-  'has_picture_img': document.querySelectorAll('picture img').length > 0
+  'has_picture_img': document.querySelectorAll('picture img').length > 0,
+
+  // Previously for 04_04.sql in 2019
+  'inline_svg_stats': (() => {
+    const svg_elements = [...document.querySelectorAll('svg')];
+
+    return {
+      total: svg_elements.length,
+      content_lengths: svg_elements.map(svg => svg.outerHTML.length),
+      props: parseNodes(svg_elements),
+    };
+  }),
+
+  // Various stats of img, source and picture elements
+  'images': (() => {
```

Comment:
This would merge in with my image property. I guess we are going with plural property names and underscores for word dividers?

I go into more detail on the state of alt and loading attributes.

Reply:
I'm generally a fan of plural names. Since it better represents it contains many resources.

As for if we should be using snake_case or dashes etc... I don't believe we ever formally decided on a standard. I generally prefer snake_case because its much easier to read and query.

Any thoughts @rviscomi?

Once we decide, lets wait until this PR is merged to make any style fixes to the rest of the file though.

Reply:
I've already converted to snake_case, so lets stick with that. I've used plurals where appropriate.